chirp/drivers/radtel_rt620.py

```diff
@@ -0,0 +1,545 @@
+                          RadioSettingValueList(
+                              self.LIST_100ms,
+                              current_index=_mem.settings.taildelay))
+        extra.append(rs)
+        rs = RadioSetting("settings.fminterrupt", "FM radio Interruption",
+                          RadioSettingValueBoolean(_mem.settings.fminterrupt))
+        extra.append(rs)
+        # Settings menu 21
+        rs = RadioSetting("settings.alarmmode", "Alarm Mode",
+                          RadioSettingValueList(
+                              self.LIST_ALARMMODE,
+                              current_index=_mem.settings.alarmmode))
+        extra.append(rs)
+        rs = RadioSetting("settings.alarmtone", "Sound Alarm",
+                          RadioSettingValueBoolean(_mem.settings.alarmtone))
+        extra.append(rs)
+        rs = RadioSetting("settings.menuquittime", "Menu Quit Timer",
+                          RadioSettingValueList(
+                              self.LIST_MENU_QUIT_TIME,
+                              current_index=_mem.settings.menuquittime))
+        extra.append(rs)
+        if self._has_pilot_tone:
+            rs = RadioSetting("settings.tone", "Pilot Tone",
+                              RadioSettingValueList(
+                                  self.LIST_PILOT_TONE,
+                                  current_index=_mem.settings.tone))
+            extra.append(rs)
+        # Settings menu 27
+        if _mem.settings.sk1short >= len(self.LIST_KEYS):
+            val = 0x00
+        else:
+            val = _mem.settings.sk1short
+        rs = RadioSetting("settings.sk1short", "Side Key 1 Short Press",
+                          RadioSettingValueList(
+                              self.LIST_KEYS, current_index=val))
+        extra.append(rs)
+        # Settings menu 28
+        if _mem.settings.sk1long >= len(self.LIST_KEYS):
+            val = 0x00
+        else:
+            val = _mem.settings.sk1long
+        rs = RadioSetting("settings.sk1long", "Side Key 1 Long Press",
+                          RadioSettingValueList(
+                              self.LIST_KEYS, current_index=val))
+        extra.append(rs)
+        # Settings menu 29
+        if _mem.settings.sk2short >= len(self.LIST_KEYS):
+            val = 0x00
+        else:
+            val = _mem.settings.sk2short
+        rs = RadioSetting("settings.sk2short", "Side Key 2 Short Press",
+                          RadioSettingValueList(
+                              self.LIST_KEYS, current_index=val))
+        extra.append(rs)
+        # Settings menu 30
+        if _mem.settings.sk2long >= len(self.LIST_KEYS):
+            val = 0x00
+        else:
+            val = _mem.settings.sk2long
+        rs = RadioSetting("settings.sk2long", "Side Key 2 Long Press",
+                          RadioSettingValueList(
+                              self.LIST_KEYS, current_index=val))
+        extra.append(rs)
+        if self._has_top_key:
```

Comment:
This is always `True` so you don't need this. Are you planning or aware of some other radio that will subclass this and set this to be `False`?

Reply:
This is actually always `False`, but the official CPS for this radio sets top keys. However, the model of the radio is not equipped with the key. I was suspecting the non-GPS version or some other radio with the same CPS has it, so   I implemented it.

Reply:
This is actually the GPS version (should this be added to model name then?), I don't have access to non-GPS one unfortunately.

Reply:
Nah, let's just leave it as it is and deal with that later if needed.